examples/get-started/first_figure.py

```diff
@@ -0,0 +1,159 @@
+===========================
+
+This tutorial page covers the basics of creating a figure. It will only use
+the ``coast`` module for plotting. Later examples will address other PyGMT
+modules.
+"""
+
+###############################################################################
+# Setting up the development environment
+# --------------------------------------
+#
+# PyGMT can be used in both a Python script and a notebook environment, such
+# as Jupyter. The tutorial's recommended method is to use a notebook, and the
+# code will be for a notebook environment. If you use a Python script instead,
+# there will be a difference in how the figure is shown that is explained in
+# the first example.
+
+
+###############################################################################
+# Loading the library
+# -------------------
+#
+# The first step is to import ``pygmt``. All modules and figure generation is
+# accessible from the :mod:`pygmt` top level package.
+
+# sphinx_gallery_thumbnail_number = 4
+import pygmt
+
+###############################################################################
+# Creating a figure
+# -----------------
+#
+# All figure generation in PyGMT is handled by the :class:`pygmt.Figure` class.
+# Start a new figure by creating an instance of this class:
+
+fig = pygmt.Figure()
+
+###############################################################################
+# To add to a plot object (``fig`` in this example), the PyGMT module is used
+# as a method on the class. This example will use the module ``coast``, which
+# can be used to create a map without any other modules or external data. The
+# ``coast`` module plots the coastlines, borders, and bodies of water using a
+# database that is included in GMT.
+#
+# First, a region for the figure must be selected. This example will plot some
+# of the coast of Maine in the northeastern US. A Python list can be passed to
+# the ``region`` argument with the minimum and maximum X-values (longitude)
+# and the minimum and maximum Y-values (latitude). For this example, the
+# minimum (bottom left) coordinates are N43.75 W69 and the maximum (top right)
+# coordinates are N44.75 W68. Negative values can be passed for latitudes in
+# the southern hemisphere or longitudes in the western hemisphere.
+#
+# In addition to the region, an argument needs to be passed to ``coast`` to
+# tell it what to plot. In this example, ``coast`` will be told to plot the
+# shorelines by passing the Boolean value ``True`` to the ``shorelines``
+# parameter. The ``shorelines`` parameter has other options for finer control,
+# but setting it to ``True`` uses its default values.
+
+fig.coast(region=[-69, -68, 43.75, 44.75], shorelines=True)
+
+###############################################################################
+# To see the figure, call :meth:`pygmt.Figure.show`. If you are using a Python
+# script instead of a notebook, use ``fig.show(method="external)`` to display
+# the figure.
```

Comment:
> If you are using a Python script instead of a notebook, use ``fig.show(method="external)`` to display the figure.

After PR #529, `fig.show()` should automatically open the preview image using an external PDF viewer when running a Python script. Thus `fig.show(method="external)` is no longer required.